webDemo/src/App.tsx

```diff
@@ -374,15 +374,15 @@ function App() {
   return (
     <View flex>
       <ScrollView>
-        <View style={styles.header}>
-          <Text style={styles.title}>Welcome to react-native-ui-lib for Web</Text>
+        <View padding-page>
+          <Text text40H center marginV-s1>Welcome to wix-react-native-ui-lib for Web</Text>
```

Comment:
The public library name doesn't include `wix` 
```suggestion
          <Text text40H center marginV-s1>Welcome to react-native-ui-lib for Web</Text>
```